data/transactions/logic/eval.go

```diff
@@ -4338,6 +4476,26 @@ func opAcctParamsGet(cx *EvalContext) error {
 		value.Uint = account.MinBalance(cx.Proto).Raw
 	case AcctAuthAddr:
 		value.Bytes = account.AuthAddr[:]
+
+	case AcctTotalNumUint:
```

Comment:
I guess that's a fair criticism, but these new fields were only published because of the new Box fields (at the bottom). The box fields had to be exposed, because they have important use cases (an app wants to know if its box state has been cleaned up yet).  The other fields were added because once we exposed the box fields, we "might as well" expose it all and get it over with.

So a PR directly to master for just the other fields would have seemed strange - on their own, the fields aren't very important to expose.